www/src/views/community/badge.js

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+
+import { rhythm } from "../../utils/typography"
+import { colors } from "../../utils/presets"
+
+const Badge = ({ forHire, children, customCSS }) => (
+  <div
+    css={[
+      styles.badge,
+      forHire ? styles.forHire : styles.hiring,
+      { letterSpacing: customCSS ? `0.025em` : null },
+      customCSS && forHire ? customCSS : {},
+    ]}
+  >
+    {children}
+  </div>
+)
+
+export default Badge
+
+const styles = {
+  badge: {
+    borderRadius: 20,
+    padding: `0 ${rhythm(1 / 3)}`,
+  },
+  hiring: {
+    background: colors.ui.light,
+    color: colors.lilac,
+  },
+  forHire: {
+    background: `#effaef`,
+    color: colors.success,
+  },
+}
```

Comment:
Are we defining styles objects at the bottom of the file? I know we talked about this at some point @jlengstorf but I don't remember. Either way, does not block merge.

Reply:
I think I adopted it from @kkemple during .com dev, and have been doing it in a couple of places on `www`, too.